gametracker_importer.lua

```diff
@@ -1,75 +1,109 @@
 -- Gametracker importer by Zerf (STEAM_0:0:46161927)
 --[[ Example:
 concommand.Add("gametracker_try", function(srvcon, cmd, args)
-	GTImport.Run("66.150.188.121:27015", 8, 1, function(mastertbl)
+	GTImport.Run("66.150.188.121:27015", 8, 3, function(mastertbl)
 		PrintTable(mastertbl)
 		file.Write("gametracker_0-400.txt", util.TableToJSON(mastertbl))
 	end)
 end)
 --]]
 local function printerr(str)
-	MsgC(Color(255, 0, 0), "[GAMETRACKER ERR]: "..str.."\n")
+	MsgC(Color(255, 0, 0), "[GAMETRACKER ERR]: " .. str .. "\n")
 end
 
 local GTImport = {}
 
 function GTImport.Run(srvip, pagecount, startpage, callback)
-	startpage = startpage or 1
 	local ret = {}
-	for i = 0, pagecount-1 do
-		local page = startpage + i
-		local url = "http://www.gametracker.com/server_info/"..srvip.."/top_players/?searchipp=50&searchpge="..page
-		http.Fetch(url, function(html, len, headers, code)
-			for i,data in pairs(GTImport.ProcessHTML(html)) do
+
+	local lastpage = pagecount + startpage - 1
+	for i = 1, pagecount do
+		local curpage = startpage + i - 1
+		local url = "http://www.gametracker.com/server_info/" .. srvip .. "/top_players/?searchipp=50&searchpge=" .. curpage
+
+		http.Fetch(url, function(html)
+			if !GTImport.ProcessHTML(html) then
+				printerr("Server not found")
+				return
+			end
+
+			for i, data in pairs(GTImport.ProcessHTML(html)) do
```

Comment:
I would recommend creating a variable to hold the result of `GTImport.ProcessHTML` so we don't have to call the function twice.